mozilla-release/mobile/android/base/java/org/mozilla/gecko/BrowserApp.java

```diff
@@ -326,6 +326,7 @@
     private SafeIntent safeStartingIntent;
     private Intent startingIntentAfterPip;
     private boolean isInAutomation;
+    private boolean hideHomeContainer;
```

Comment:
Out if the Cliqz block and doesn't have the m prefix, it is also not clearly initalized